web/blob/static/js/status.js

```diff
@@ -0,0 +1,26 @@
+/* globals PATH_PREFIX, jQuery */
+(function ($, undefined) {
```

Comment:
declare `undefined` is better for minification. I know, there is almost nothing here, but it's just my practice. About `jQuery`, it's all about protect `$` sign, again (..not *that* necessary), part of my own practice. Would you prefer to keep with the current style?